### PR TITLE
zomborg devourer

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary
Content "zomborg devourer"

#### Purpose of change

Make the game stranger and more challenging over time

#### Describe the solution

A zomborg upgrade that combines elements of zomborg and dissoluted devourer

#### Describe alternatives you've considered

Figure out the code well enough to allow it to eat robots. Give it PARROT. Give it OPERATE. Give it lore appropriate dialogue to parrot.

#### Testing

Monster spawns no errors, looks right, and behaves as expected.

#### Additional context

All honor to free-coffee for the original devourer. Any idea that good demands children.

### CODE REVIEW

**Comment by anoobindisguise** [comment]
Those parrot lines don't feel appropriate to me? seems like it shouldn't vocalize much

**Comment by LyleSY** [comment]
Really depends on where the lore is on this. I'm imagining this as being a more sentient blobby thing that has several independent pieces that can talk, mostly robots and cybernetics it's eaten with a devourer stitching it together. I'm thinking about some new lines where the pieces argue, the two most dominant personalities a cyborg mechanical brain and a nursebot.

File: data/json/monsters/cyborgs.json [hunk 1]
```diff
@@ -238,5 +239,54 @@
...
+    "armor_bash": 12,
+    "armor_cut": 28,
+    "armor_bullet": 8,
+    "bleed_rate": 25,
+    "vision_night": 3,
+    "weakpoint_sets": [ "wps_cyborg_body" ],
+    "families": [ "prof_wp_cyborg" ],
+    "harvest": "mon_zomborg",
+    "dissect": "dissect_mon_zomborg",
+    "special_attacks": [ [ "GRAB", 7 ], [ "scratch", 20 ], [ "ZOMBIE_FUSE", 80 ], [ "OPERATE", 30 ], [ "PARROT", 80 ] ],
```
Comment (TheShadowFerret) [inline data/json/monsters/cyborgs.json:274]: How will OPERATE function on a monster that isn't also PACIFIST like the defective nursebot? Isn't this just a delayed instakill?

**Comment by PatrikLundell** [comment]
Doesn't make any lore sense to me. As far as I understand, the Exodii workers are essentially slaves, rather than "free" Exodii with a will to explore. When degraded into zombies, they'd act like zombies with bionic parts, and I won't complain about them fusing, just about the research and CBM stealing parts that don't make any sense to me.

File: data/json/monsters/cyborgs.json (hunk 1 above)
Comment (LyleSY) [reply data/json/monsters/cyborgs.json:274]: Yeah, in my testing it just complains about not having any anaesthetic. It's a fun idea but it seems like this isn't the right method

Comment (LyleSY) [reply data/json/monsters/cyborgs.json]: Removing OPERATE

**Review by TheShadowFerret — APPROVED** [review]

**Pull request merged**